index.html

```diff
@@ -0,0 +1,350 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="description" content="">
```

Comment:
If you're going to use this `meta` tag, you should add content to it. If you don't want to add content to it, then remove it.